structuralcodes/core/_section_results.py

```diff
@@ -131,7 +120,7 @@ def __format__(self, spec: str) -> str:
         return output_string
 
     def __str__(self) -> str:
-        """Returns the informal string representation of the gross concrete
+        """Returns the informal string representation of the concrete
```

Comment:
same here, we should remove the concept of "concrete" now that we have a generic SectionProperty?

Reply:
Fixed in c453e63.